mitiq/tests/test_utils.py

```diff
@@ -274,3 +291,46 @@ def test_are_close_dict():
     dict2 = {"a": 1, "b": 0.0, "c": 1}
     assert not _are_close_dict(dict1, dict2)
     assert not _are_close_dict(dict2, dict1)
+
+
+def test_max_ent_state_circuit():
+    """Tests 1-qubit and 2-qubit maximally entangled states are generated."""
+    two_state = np.array([1, 0, 0, 1]) / np.sqrt(2)
+    four_state = np.array(3 * [1, 0, 0, 0, 0] + [1]) / 2.0
+    assert np.allclose(
+        _max_ent_state_circuit(2).final_state_vector(), two_state
+    )
+    assert np.allclose(
+        _max_ent_state_circuit(4).final_state_vector(), four_state
+    )
+
+
+def test_circuit_to_choi_and_operation_to_choi():
```

Comment:
Split into two tests, one for `circuit_to_choi` and the other for `operation_to_choi`.